<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: white;
}

#titleWidget{
background-color: #1d3453;
}
#title {
color: white;
}

QLineEdit{
	border:1px solid gray;
	border-radius: 6px;
	padding-left: 15px;
	height: 35px;
}

QComboBox{
border: 1px solid black;
border-radius: 8px;
background-color: #1d3453;
color: white;
height: 35px;


}

QDateEdit{
	border:1px solid gray;
	border-radius: 8px;
	padding-left: 15px;
	height: 35px;
	background-color: #1d3453;
color: white;
}</string>
  </property>
  <widget class="QPushButton" name="edit_case_info">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>110</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(0, 139, 0);
	color:white;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px;
}

QPushButton:hover{
	background-color: rgb(0, 170, 0);
}</string>
   </property>
   <property name="text">
    <string>Edit Case</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>440</width>
     <height>330</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Case ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="caseIDinput_edit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Case Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="caseNameinput_edit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Case Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="caseType_edit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Poppins</family>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Civil</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Criminal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Appeal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Class Action</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bankruptcy</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Start Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="startDate_edit">
          <property name="font">
           <font>
            <family>Poppins</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>End Date</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QLineEdit" name="endDate">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>135</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#endDate{
	width: 50px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="caseStatus_edit">
        <property name="font">
         <font>
          <family>Poppins</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Pending</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ongoing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Closed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Open</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Edit Case Information</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="edit_case_cancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: #1d3453;
	color:white;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px;
}

QPushButton:hover{
	background-color: rgb(53, 96, 153)
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
